<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>icon</class>
 <widget class="QDialog" name="icon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/logo1.png</normaloff>:/icon/logo1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 247, 241); </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>30</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116); </string>
   </property>
   <property name="text">
    <string>Choose an icon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>610</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgb(95, 167, 175), stop: 1 rgb(58, 80, 116));
color: rgb(249, 247, 241); 
font: 14pt &quot;Baskerville Old Face&quot;;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>680</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Baskerville Old Face&quot;;
color: rgb(58, 80, 116); </string>
   </property>
   <property name="text">
    <string>Icon made by Freepik from www.flaticon.com</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>311</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QLabel" name="img3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/bull.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="img8">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/ostrich.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="img2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/boar.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="img10">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/zoo.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QRadioButton" name="bull">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="deer">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="hed">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="img4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/deer.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QRadioButton" name="beaver">
      <property name="text">
       <string>(default)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QRadioButton" name="duck">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="boar">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="img9">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/owl.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="zoo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="img5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/duck.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="ostrich">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QRadioButton" name="owl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="img7">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/hippo.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="img">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/beaver.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QRadioButton" name="hippo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="img6">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/avatar/hedgehog.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
